sine_wave_publisher:
  ros__parameters:
    # How many points to publish per second
    publisher_frequency: 50.0  # in Hz and in range of [0.0, 100.0]

    # Amplitude of the sine wave (can be negative)
    amplitude: 3.0 # in range of [0.0, 50.0]

    # How fast the sine wave oscillates (rad/s)
    # 2π means one complete cycle per second
    angular_frequency: 6.28  # in range of [0.0, 62.8]. Must be positive and Limited to 10 Hz (ω = 2πf)

    # Phase shift of the wave (radians)
    # 0 means sine wave starts at 0
    # π/2 would mean it starts at amplitude
    phase: 0.0  # in radians. in range of [-6.28, 6.28] i.e Between -2π and 2π